app/src/main/res/values-pl/strings.xml

```diff
@@ -463,10 +463,17 @@
     <string name="double_button_widget_description">Widżet dwuprzyciskowy</string>
     <string name="single_button_widget_description">Widżet jednoprzyciskowy</string>
     <string name="widget_app_not_initialized">Proszę się zalogować!</string>
-    <string name="widget_command_error">Błąd komunikacji!</string>
     <string name="widget_command_no_connection">Brak połączenia z internetem!</string>
+    <string name="widget_command_error">Wystąpił błąd! (kod: %1$i)</string>
+    <string name="widget_command_connection_failure">Błąd połączenia! (kod: %1$i)</string>
+    <string name="widget_command_subject_offline">%1$s jest offline!</string>
+    <string name="widget_command_subject_unavailable">%1$s nie istnieje!</string>
+    <string name="widget_command_no_access">Brak dostępu! (code: %1$i)</string>
     <string name="widget_command_started">Wykonano</string>
     <string name="widget_removed_label">Kanał usunięty</string>
+    <string name="widget_group">grupa</string>
```

Comment:
Scene / Group should not be capitalized if inserted at the beginning of a sentence?

Reply:
I'm capitalizing programmatically.